rllib/examples/custom_tf_policy.py

```diff
@@ -20,13 +21,22 @@ def policy_gradient_loss(policy, model, dist_class, train_batch):
     logits, _ = model.from_batch(train_batch)
     action_dist = dist_class(logits, model)
     return -tf.reduce_mean(
-        action_dist.logp(train_batch["actions"]) * train_batch["rewards"])
+        action_dist.logp(train_batch["actions"]) * train_batch["advantages"])
+
+
+def calculate_advantages(policy,
+                         sample_batch,
+                         other_agent_batches=None,
+                         episode=None):
+    sample_batch["advantages"] = discount(sample_batch["rewards"], 0.99)
```

Comment:
this should try to fetch gamma from policy config instead right

Reply:
Not for the example.